proxy/src/control_plane/client/neon.rs

```diff
@@ -134,8 +134,8 @@ impl NeonControlPlaneClient {
                 project_id: body.project_id,
             })
         }
-        .map_err(crate::error::log_error)
-        .instrument(info_span!("http", id = request_id))
+        .inspect_err(|e| tracing::debug!(error = ?e))
+        .instrument(info_span!("do_get_auth_info", request_id))
```

Comment:
can we also remove the request_id? It's no longer used (we used to send it to cplane, but now we use session_id)

Reply:
yes, sure. The req id is not the sess id, so was wondering why we have it as "id" again.

Reply:
Done.